repos/system_upgrade/common/actors/initramfs/enable_lvm_autoactivation/actor.py

```diff
@@ -0,0 +1,21 @@
+from leapp.actors import Actor
+from leapp.libraries.actor import enable_lvm_autoactivation as enable_lvm_autoactivation_lib
+from leapp.models import UpgradeInitramfsTasks
+from leapp.tags import FactsPhaseTag, IPUWorkflowTag
+
+
+class EnableLVMAutoactivation(Actor):
+    """
+    Enable LVM autoactivation in upgrade initramfs.
+
+    Produce instructions for upgrade initramfs generation that will result in LVM
+    autoactivation in the initramfs.
+    """
+
+    name = 'enable_lvm_autoactivation'
+    consumes = ()
+    produces = (UpgradeInitramfsTasks, )
+    tags = (FactsPhaseTag, IPUWorkflowTag)
+
+    def process(self):
+        enable_lvm_autoactivation_lib.emit_lvm_autoactivation_instructions()
```

Comment:
Thinking about this actor, it's executed unconditionally. Do we want to make this happen on systems without LVM and in case the livemode is used? not a blocker, just making a note to think about it